src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
i think in this case better to use enum

Reply:
Why, could you please explain?

Reply:
Agree with @AntonHerasymov. @Orpheus29 In this case you will be able to use values of the enum(`Shape.triangle`...) when you set value for your shape inside class(and avoid using "magic values")
For example here:

<img width="394" alt="image" src="https://github.com/mate-academy/ts_figures_classes/assets/86789161/fd096871-a8f0-4800-b9fd-11b99d61ccbf">

You will be able to do like this -> `public shape: Shape = Shape.triangle;`